pkg/sdn/plugin/common.go

```diff
@@ -171,3 +172,28 @@ func RunNamespacedPodEventQueue(client kcache.Getter, namespace string, closeCha
 		}
 	}
 }
+
+func getDeletedObjFromInformer(obj interface{}, resourceName ResourceName) (interface{}, error) {
+	var expectedObjType interface{}
+
+	switch resourceName {
+	case Nodes:
```

Comment:
You call this func with 4 different ResourceName values (Namespaces, Services, Pods, Nodes) but only support Nodes?